### PR TITLE
update policy docs

### PR DESCRIPTION
**Pull request by Ghostvv**
**Proposed changes**:
- update description of the policies

**Status (please check what you already did)**:
- [ ] made PR ready for code review
- [ ] added some tests for the functionality
- [x] updated the documentation
- [ ] updated the changelog

### CODE REVIEW

File: docs/policies.rst [hunk 1]
```diff
@@ -54,55 +59,64 @@ In pseudocode, what the ``SimplePolicy`` above does is:
...
 
-.. literalinclude:: ../rasa_core/agent.py
-   :pyobject: Agent.train
+    def train(self,
+          training_trackers,  # type: List[DialogueStateTracker]
+          domain,  # type: Domain
+          **kwargs  # type: **Any
+          ):
+    # type: (...) -> None
+    """Trains the policy on given training trackers."""
```
Comment (tmbo) [inline docs/policies.rst]: this function is missing a code block, should be at least `pass`

Comment (Ghostvv) [reply docs/policies.rst]: in the superclass it is `raise NotImplementedError("Policy must have the capacity to train.")`

What is better `pass` or `raise`?

Comment (tmbo) [reply docs/policies.rst]: mhm lets use the raise

Comment (Ghostvv) [reply docs/policies.rst]: then in predict_action_probabilities as well?
I will use the code directly from methods from `policy.py`

**Pull request merged**